exec/META

```diff
@@ -0,0 +1,11 @@
+package "freespec-exec" (
+
+    description = "Freespec Exec Plugin"
+    version     = "8.10"
```

Comment:
Is this the supported version of Coq?

If so, the current version is 8.9, right?

Reply:
Yes, this file came from development version of Coq.
Fix by @2d19f1323fb29c9cf1b39268901a0e77077fde08